### PR TITLE
Admin Res Data Pull From FB

### LINKED ISSUE #57: Add functionality to the reservations page

### LINKED ISSUE #91: Update the styling for the v-data-table in AdminActiv and AdminLog

### PR DESCRIPTION
**Pull request by dperezd**
Files edited: AdminActive.vue, AdminLog.vue, and techCenterCheckoutData.js

Upcoming Reservations and Reservations Log table now pull data from firebase.
-created two new functions in techCenterCheckoutData.js (getAdminCurrent, getResLog)
-created new method in AdminActive.vue and AdminLog.vue

### CODE REVIEW

**Review by greyhixson — CHANGES_REQUESTED** [review]
Code looks good, but the date isn't being displayed properly:
image

**Comment by dperezd** [comment]
I changed timestamp error in techCenterCheckout.Data.js

**Review by greyhixson — CHANGES_REQUESTED** [review]
Better but what's the '3991', and it's wordy to say Central Daylight Time or Central Standard Time. A lot of extra detail is in the date that should be hidden.
image
Also your branch is out of date with master

**Comment by dperezd** [comment]
I changed the data table in both pages to lists (Tables didn't feel necessary since admin will most likely not be modifying these). I have still not been able to resolve the date display issue, I haven't found a good way to transition from timestamps to dates to strings. I am still working out of this same branch.

File: src/components/Admin/AdminLog.vue [hunk 1]
```diff
@@ -1,102 +1,78 @@
...
+          <v-list-item-subtitle v-text="'Reservation ID: ' + device.reservationID" />
+
+          <v-list-item-subtitle v-text="'Device Tag: ' + device.deviceTag" />
+
+          <v-list-item-subtitle v-text="'Username: ' + device.username" />
+
+          <v-list-item-subtitle v-text="'First Name: ' + device.firstName" />
+
+          <v-list-item-subtitle v-text="'Last Name: ' + device.lastName" />
+
```
Comment (greyhixson) [inline src/components/Admin/AdminLog.vue]: lots of unnecessary spacing
```suggestion
          <v-list-item-subtitle v-text="'Reservation ID: ' + device.reservationID" />
          <v-list-item-subtitle v-text="'Device Tag: ' + device.deviceTag" />
          <v-list-item-subtitle v-text="'Username: ' + device.username" />
          <v-list-item-subtitle v-text="'First Name: ' + device.firstName" />
          <v-list-item-subtitle v-text="'Last Name: ' + device.lastName" />
```

**Review by greyhixson — CHANGES_REQUESTED** [review]
Out of date with master and a small spacing thing. Besides that good job!

**Review by greyhixson — APPROVED** [review]

**Pull request merged**